reference-implementation/to-upstream-wpts/writable-streams/write.js

```diff
@@ -197,3 +197,15 @@ promise_test(() => {
           assert_equals(writeCount, numberOfWrites, `should have called sink's write ${numberOfWrites} times`));
   });
 }, 'a large queue of writes should be processed completely');
+
+promise_test(() => {
+  const stream = recordingWritableStream();
+  const w = stream.getWriter();
+  const WritableStreamDefaultWriter = w.constructor;
+  w.releaseLock();
+  const writer = new WritableStreamDefaultWriter(stream);
```

Comment:
Hmm, this looks something we didn't intend to enable. We could disallow this by letting the constructor to look at some internal slot of the stream which can be set only by getWriter(), e.g. WDYT, @domenic ?